libraries/platforms/dromajo-vcs/bp_hb_platform.h

```diff
@@ -0,0 +1,96 @@
+// BlackParrot platform-specific header file
+// This file contains manycore-specific platform functions and constants
+
+#ifndef BP_HB_PLATFORM_H
+#define BP_HB_PLATFORM_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <bsg_manycore.h>
+#include <bsg_manycore_epa.h>
+
+// Host X and Y coordinates
+#ifndef HOST_X_COORD
+#define HOST_X_COORD 15
+#endif
+
+#ifndef HOST_Y_COORD
+#define HOST_Y_COORD 8
+#endif
+
+// BlackParrot coordinates
+// Note: Currently not all of these coordinates are in use
+// but might be useful during BlackParrot integration
+#ifndef BP_COORDS
+#define BP_POD_X 0
+#define BP_POD_Y 1
+// Coordinates of BP-HB bridge used for manycore communication
+#define BP_HOST_LINK_X 15
+#define BP_HOST_LINK_Y 9
+// Coordinates of BP-HB bridge used for DRAM loads and stores
+#define BP_DRAM_0_LINK_X 15
+#define BP_DRAM_0_LINK_Y 10
+#define BP_DRAM_1_LINK_X 15
+#define BP_DRAM_1_LINK_Y 11
+#endif
+
```

Comment:
Instead of setting defaults, error. For the host coordinate in particular, it can and should be set from the make flow.

Reply:
I added #error statements for the host coordinates. For BlackParrot coordinates I left them as is. I think we can revisit it when BlackParrot integration happens because there are multiple BPs in different pods and stuff so this most likely would change?

Reply:
I think having the Host/DRAM ones be constant is fine. My understanding is that POD_X/Y might change even more than the HOST coordinates.

Let's make the HOST/DRAM ones constant (outside of the IFNDEF). Then have the BP_POD_X/Y ones set by the make flow. Error out if not set.

Reply:
Done